### DISCUSSION TITLE
How to create turborepo without typescript?

### DISCUSSION THREAD

**Discussion by stevenFrozt** [vercel/turborepo#3153] [Help] [answered]
is there any template no typescript and with tailwind ?

**Comment by mehulkar** [comment, marked as answer]
We don't have an `example` for this, but you should be able to get pretty far with the [`with-tailwind`](https://github.com/vercel/turbo/tree/main/examples/with-tailwind) example and remove the typescript config and `packages/tsconfig` from it. Alternatively, you can start a new repository however you want and [_add_ turborepo to it](https://turbo.build/repo/docs/getting-started/existing-monorepo)
